platform_versioned_docs/version-24.2/cli/commands.mdx

```diff
@@ -322,9 +322,9 @@ Manage [Studio sessions](../data_studios/index.mdx) hosted in Seqera Platform. S
   Description         |                                                            
   Created on          | Fri, 31 Jan 2025 18:12:27 GMT                              
   Created by          | rob-newman | rob.newman@seqera.io                          
-  Template            | public.cr.seqera.io/platform/data-studio-jupyter:4.1.5-0.7 
+  Template            | public.cr.seqera.io/platform/studio-jupyter:4.1.5-0.7 
```

Comment:
I don't think we should make these changes yet. Those are real files and deleting the `data-` prefix isn't reality.